src/App/App.tsx

```diff
@@ -108,6 +109,13 @@ const ConnectedApp: FC<ConnectedAppProps> = ({
         action: open.openLauncher,
     });
 
+    useHotKey({
+        hotKey: 'ctrl+r',
+        title: 'Restart app',
+        isGlobal: true,
+        action: () => getCurrentWindow().reload(),
```

Comment:
To make this work the same as the current ctrl/cmd-r shortcut, this should be changed:
```suggestion
        action: () => getCurrentWindow().emit('restart-window'),
```
(The behaviour seems mostly the same but there seems to be some difference)